databases/libpqxx/Portfile

```diff
@@ -52,13 +52,17 @@ if {!$any_variant_set} {
     set server postgresql16
 }
 
-depends_build   port:pkgconfig
+depends_build   path:bin/pkg-config:pkgconfig
 
 if {${os.platform} eq "darwin" && ${os.major} < 20} {
     depends_build-append  port:python311
+
     post-patch {
-        reinplace -W ${worksrcpath} "s| /usr/bin/env python3|${prefix}/bin/python3.11|" \
-            tools/splitconfig tools/template2mak.py tools/test_all.py tools/m4esc.py
+        fs-traverse f ${worksrcpath} {
+            if {[string match *.py ${f}]} {
+                reinplace -q "s| /usr/bin/env python3|${prefix}/bin/python3.11|" ${f}
```

Comment:
Would it be better to make this (and the dependency above) a variable so that when the Python version is updated, the scripts get updated too?

Reply:
Yeah, sure, we can do that. Will update once back to the laptop.

Reply:
can this be changed to Python 3.12 as well - that has been MacPorts' default for about 8 months now

Reply:
> can this be changed to Python 3.12 as well - that has been MacPorts' default for about 8 months now

That is neither necessary nor, IMO, desirable, but w/e, done.